feat(create-vite): support create-vite-extra

bluwy opened the pull request:
<!-- Thank you for contributing! -->

### Description

Add `create-vite-extra` under new `Others` option in framework selection:

```
✔ Project name: … vite-project
✔ Select a framework: › Others
? Select a variant: › - Use arrow-keys. Return to submit.
❯   create-vite-extra ↗
```

### Additional context

<!-- e.g. is there anything you'd like reviewers to focus on? -->

---

### What is the purpose of this pull request? <!-- (put an "X" next to an item) -->

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

bluwy committed:
feat(create-vite): support create-vite-extra

patak-dev commented:
I think this is a good idea, allowing folks to reach out for a curated list of templates without adding extra burden on the Vite Core repo. Let's discuss it in today's meeting.

Do you know why `Others` doesn't get underlined when selected? Not a big deal, it was just a bit surprising when I tested it.

bluwy commented:
Seems like using the `reset` color removes the underline, all perhaps we can choose a different color.

bluwy committed:
chore: move under others

bluwy committed:
feat: add arrows for external cli

bluwy committed:
chore: merge main

bluwy commented:
From last meeting's discussion, I've moved `create-vite-extra` under the new `Others` option in the framework selection step. I also added `↗` for options that open a new CLI.
